projects/etos_suite_runner/src/etos_suite_runner/lib/esr_parameters.py

```diff
@@ -155,19 +155,34 @@ def tercc(self) -> Union[list[dict], dict]:
             self.etos.config.set("tercc", tercc)
         return self.etos.config.get("tercc")
 
+    def _get_test_suite_list_from_tercc(self) -> list[Suite]:
+        """Read test suite list from TERCC."""
+        tercc = json.loads(os.getenv("TERCC", "{}"))
+        self.logger.info("Reading test suites from TERCC: %s", tercc)
+        if isinstance(tercc, list):
+            test_suite = [Suite(**suite) for suite in tercc]
```

Comment:
This part was the part that handled TESTRUN before. It shouldn't be needed here.